controllers/storageautoscaler/verify_test.go

```diff
@@ -0,0 +1,27 @@
+package storageautoscaler
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestFailIfTimeOut(t *testing.T) {
+	timeoutSeconds := 100
```

Comment:
Does this unit test run for 100 seconds? If so, that's way too long for unit tests. When I do unit tests for timeouts, I usually modify the values to run with timeouts on the order of 10 milliseconds. It might be necessary to have the function accept timeouts on the millisecond scale simply to help with the unit testing